### PR TITLE
Fixes Hijack Objective

### PR DESCRIPTION
**Pull request by Fox-McCloud**
Fixes hijack so multiple tator-tots can actually work together.

ERT exclusion actually works now.

### CODE REVIEW

File: code/game/gamemodes/objective.dm [hunk 1]
```diff
@@ -339,7 +339,7 @@ datum/objective/hijack
 			return 0
 		var/area/shuttle = locate(/area/shuttle/escape/centcom)
 		for(var/mob/living/player in player_list)
-			if(istype(player, /mob/living/silicon) || istype(player, /mob/living/simple_animal) || player.mind.special_role && !player.mind.special_role == "Response Team")
+			if(istype(player, /mob/living/silicon) || istype(player, /mob/living/simple_animal) || player.mind.special_role)
```
Comment (TheDZD) [inline code/game/gamemodes/objective.dm]: How the heck was that simple of a check managing to break?

Comment (Fox-McCloud) [inline code/game/gamemodes/objective.dm]: No idea, but it was; tested this specifically with a couple of people last night---had two traitors on the shuttle? Failed. Traitor+mindslaved? Failed.

Removed the ERT check->Two traitors? Success. Traitor+Mindslaved? Success.

Comment (Crazylemon64) [inline code/game/gamemodes/objective.dm]: I think the ! operator has higher precedence than the == operator - either replace the == with !=, or put the entire check besides the ! inside parentheses, like so: !(player.mind.special_role == "Response Team")

Comment (TheDZD) [inline code/game/gamemodes/objective.dm]: That's probably it.

**Pull request merged**